CHANGELOG.md

```diff
@@ -225,6 +225,8 @@ The format is based on [Keep a Changelog].
 -   Added support for parameterized circuits. (\#2103)
 -   Added preset PassManagers that offer predetermined pipelines of
     transpiler passes. (\#2163)
+-   Possible to decompose SU(4) gate into non-CNOT basis with
+    `TwoQubitDecomposer`
```

Comment:
This doesn't seem related to this PR, probably just a rebase or merge issue.

Reply:
no it was in the wrong place (under the "fixed" heading). I moved it under the "added" heading, and also it was for an older release not current.

Reply:
This line is not about adding the `TwoQubitBasisDecomposer`, it was about a fix made to it post 0.8.0. It was added in: https://github.com/Qiskit/qiskit-terra/pull/2546 and it was about the fix being made there. (the entry is just poorly worded making this hard to figure out). The changelog entry added with the PR adding he `TwoQubitBasisDecomposer` is also not super clear, it's the first bullet here: https://github.com/Qiskit/qiskit-terra/blob/master/CHANGELOG.md#added-1 which was added in: https://github.com/Qiskit/qiskit-terra/pull/2121

But, regardless this doesn't change my point that changing this is unrelated to this PR. If we want to make independent changes to the changelog, we should do that in a separate PR to isolate this unrelated discussion from talking about the contents of this patch. Doing this separately will also makes the git log cleaner for potential revert and bisect debugging in the future.

Reply:
ok i fixed it